docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc

```diff
@@ -45,6 +45,11 @@ The default cron expression runs every 24 hrs at 12:00 a.m.
   One can use `ssh-keygen -l -f <(ssh-keyscan $TARGET)` to find the fingerprints of a remote system.
 | _unchecked_
```

Comment:
```suggestion
| n/a
```

Reply:
I don't think `n/a` is a good match. The default value really means, that the host key is not checked.

Reply:
Is there a default value, or is this more of a default action? Would it be clearer to say "By default, the monitor does not check the host key."